packages/bbui/src/ActionMenu/ActionMenu.svelte

```diff
@@ -23,7 +23,7 @@
   // This is needed because display: contents is considered "invisible".
   // It should only ever be an action button, so should be fine.
   function getAnchor(node: HTMLDivElement) {
-    anchor = (node.firstChild as HTMLElement) ?? undefined
+    anchor = (node.firstElementChild as HTMLElement) ?? undefined
```

Comment:
Needed this for the ActionMenu to work correctly because in Svelte 5 slotted content sometimes gets wrapped